ios/MaasNearbyTransit/Tests/MaasNearbyTransitTests/XCTestManifests.swift

```diff
@@ -0,0 +1,9 @@
+import XCTest
+
+#if !canImport(ObjectiveC)
+public func allTests() -> [XCTestCaseEntry] {
+    return [
+        testCase(MaasNearbyTransitTests.allTests),
```

Comment:
I'm not sure it makes sense to have dummy test cases if they're not used.

Reply:
I think at the moment non of our feature packages should include `Test` target. Deleting this 🔥